docs/cookbook/sylius-cloud/basic-project-setup.rst

```diff
@@ -0,0 +1,163 @@
+Basic project setup
+===================
+
+Sylius is a typical Symfony application, so all the steps below can be used for a clean Symfony application as well.
+
+Prerequisites
+-------------
+
+To work with Sylius Cloud you'll need the `platform` (CLI) command on your computer. To install it, please run the script:
+
+.. code-block:: bash
+
+    curl -fsSL https://raw.githubusercontent.com/platformsh/cli/main/installer.sh | bash
+
+There are several methods of installing the `platform` application in your machine. To see them, please visit the `documentation <https://docs.platform.sh/administration/cli.html#1-install>`_
+
+Pushing Sylius to Sylius Cloud
+------------------------------
+
+Let's assume you're extending the Sylius-Standard in version including the Sylius Cloud configuration.
+
+First step you need to do is to create a Sylius Cloud project. To do it, please log in to your Sylius Cloud account and click "Create a new project" button from the dashboard.
+You can also do it by running the CLI command:
+
```

Comment:
In this case I think we should mention that we need to log in to Platform.sh account, as it could be misleading a bit 🤔 
```suggestion
First step you need to do is to create a Sylius Cloud project. To do it, please log in to your Platform.sh account or create a new one if you have not any and click "Create a new project" button from the dashboard.
You can also do it by running the CLI command:

```

Reply:
I would also mention creating the organisation, as without it it's impossible to create a project

Reply:
And add some screenshots, simplyfing the first steps